src/state_machine.zig

```diff
@@ -1140,10 +1130,10 @@ test "create/lookup/rollback accounts" {
                     .debits_must_not_exceed_credits = true,
                     .credits_must_not_exceed_debits = true,
                 },
-                .debits_pending = math.maxInt(u64),
-                .debits_posted = math.maxInt(u64),
-                .credits_pending = math.maxInt(u64),
-                .credits_posted = math.maxInt(u64),
+                .debits_pending = 1,
+                .debits_posted = 1,
+                .credits_pending = 1,
+                .credits_posted = 1,
```

Comment:
Set these to `0` as an invalid value intentionally, also for any other test vectors where the expected error code should be returned before the account balances are checked.

We already do this for everything, so that the tests fail also if any business logic is re-ordered to return errors in different order.

Reply:
Ah, my brain was working upside down.

Reply:
My comment is struck from the record. 😬